ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -715,11 +716,37 @@ export default defineComponent({
     }
 
     const scrollToFirstError = async (scrollToTop: boolean = false): Promise<void> => {
+      const validationSections = [
+        'staff-transfer-submitting-party',
+        'transfer-ref-num-section',
+        'transfer-confirm-section',
+        'transfer-certify-section',
+        'staff-transfer-payment-section'
+      ]
+      const sectionValidationStates = [
+        localState.isSubmittingPartyValid,
+        isRefNumValid.value,
+        localState.isCompletionConfirmed,
+        !localState.validateAuthorizationError,
+        !localState.validateStaffPayment
+      ]
+      // Map the review sections to their true/false values
+      const mappedSections = validationSections.map(function (key, val) {
+        return { section: key, value: sectionValidationStates[val] }
+      })
+      // If not staff transfer flow, remove the sections that wont be present
+      if (!isRoleStaffReg.value) {
+        mappedSections.shift()
+        mappedSections.pop()
+      }
+      // Set the first invalid section
+      const invalidSection = document.getElementById(mappedSections.filter(item => item.value === false)[0].section)
+
       setTimeout(() => {
         scrollToTop
           ? document.getElementById('mhr-information-header').scrollIntoView({ behavior: 'smooth' })
           : document.getElementsByClassName('border-error-left').length > 0 &&
-            document.getElementsByClassName('border-error-left')[0].scrollIntoView({ behavior: 'smooth' })
+            invalidSection.scrollIntoView({ block: 'start', inline: 'nearest', behavior: 'smooth' })
```

Comment:
I'm not sure this is required.

What is the requirement/issue for this code?

Reply:
The autoscroll was scrolling to the top of the components, not the section. This enables it to scroll to where the number header is. Ill pm you some screenshots